packages/api/src/external/carequality/__tests__/cq-directory.ts

```diff
@@ -0,0 +1,33 @@
+import { CQDirectoryEntry } from "../cq-directory";
+import { makeBaseDomain } from "../../../domain/__tests__/base-domain";
+import { faker } from "@faker-js/faker";
+import { Organization } from "@metriport/carequality-sdk/models/organization";
+
+// TODO implement this
+export function makeOrganization(): Organization | undefined {
```

Comment:
No, intentionally left like this. Just needed some object that returned the shape needed, and this is for testing purposes only. This PR is already a stretch since it was not planned, so I didn't want to spend more time on it if not needed.